var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -482,6 +482,11 @@ def setup_build_environment(self, env):
 
         env.set("NPY_LAPACK_ORDER", lapack)
 
+    def setup_run_environment(self, env):
+        archs = ("x86_64_v4:", "cannonlake:", "mic_knl")
```

Comment:
Can you add the URLS of the github issues/discussions as hyperlinks above the `archs =` line`, for example:
```
    # https://linktomatplotlibissue
    # https://linktonumpyissue
    archs = ...
```

Thanks!

Reply:
Done, thanks!